test/dlc_tests/sanity/test_pre_release.py

```diff
@@ -643,6 +659,43 @@ def _assert_artifact_free(output, stray_artifacts):
             artifact, output.stdout
         ), f"Matched {artifact} in {output.stdout} while running {output.command}"
 
+def _test_sm_toolkit_and_ts_version(image, region):
+    """
+    @param image: ECR image URI
+    """
+
+    if "gpu" or "cpu" in image:
+        cmd_smkit = "pip show sagemaker-pytorch-inference | grep -i Version"
+        cmd_ts = "torchserve --version"
+    else:
+        ValueError(
+            "Test as of now only covers CPU and GPU type images. If required, please modify this test to accommodate the new image type!")
+
+    ctx = Context()
+    container_name = get_container_name(
+        "pytorch-smtoolkit-ts-check", image)
+    start_container(container_name, image, ctx)
+    try:
+        output_smkit = run_cmd_on_container(
+            container_name, ctx, cmd_smkit, executable="bash"
+        )
+        toolkit_version_from_output = (
+            (str(output_smkit.stdout).split(' '))[1]).strip()
```

Comment:
can you provide the expected output and whether this slicing will scale? It probably will be more robust to do a regex search here

Reply:
changed to regex